docs/UserGuide.md

```diff
@@ -116,13 +116,18 @@ Format: `list`
 
 Adds a patient into HealthSync, with the given patient information.
 
-* A patient's name and IC are required when creating a new entry into HealthSync.
-* All the compulsory fields must be provided.
-* Optional fields like appointment and medical history need not be provided.
+* A patient's [name](#name) and [ID](#ID) are required when creating a new entry into HealthSync.
+* If the name or ID already exist in HealthSync, you cannot add the patient into HealthSync.
+* All the compulsory [fields](#fields) must be provided.
+* Optional fields like [Appointment](#appointment) and [medical history](#medical-history) need not be provided.
 
-Format: `add n/NAME id/ID_NUMBER [field] ...`
+[Format](#command-format):
+`add n/NAME id/ID_NUMBER p/PHONE_NUMBER e/EMAIL_ADDRESS a/ADDRESS [m/MEDICAL_HISTORY]... [ap/APPT]`
 
->:bulb: Use `a` as a shortcut for `add`
+> :bulb: Use `a` as a shortcut for `add`
+
+> :exclamation: Currently, you are not allowed to add a duplicate name into the app! This is an enhancement that
+> we plan to add to the program in the future.
```

Comment:
Should `app` and `program` be changed to `HealthSync` instead

Reply:
This message has been adjusted into a different form, so this is no longer an issue.